shared/xccdf/system/software/disk_partitioning.xml

```diff
@@ -170,6 +170,9 @@ Any <i>PASSPHRASE</i> is stored in the Kickstart in plaintext, and the Kickstart
 Omitting the <tt>--passphrase=</tt> option from the partition definition will cause the
 installer to pause and interactively ask for the passphrase during installation.
 <br /><br />
+It is recommended to use at least the <tt>aes-xts-plain64</tt> cipher with
+a minimum <tt>512</tt> bit key size.
```

Comment:
Phrasing "recommended to use at least" means the user must express judgement over the algorithms.... and leads to questions over what's sufficient/deficient. Most people don't know how to judge this. Heck, I don't.... wtf is AES XTS?

Instead, it may be better to state what the minimum standard is and provide a link to where users can read about other ciphers and their tradeoffs.

e.g.
``The aes-xts-plain64 cipher, with a 512 bit key, reflects minimally accepted standards. Details regarding additional ciphers and their tradeoffs can be found at $fooURL``

Reply:
@shawndwells made updates. I defer to the LUKS guide that is linked below this sentence for the cipher evaluation. Do we have an official document that talks about ciphers and their tradeoffs? If not, we probably are going to have to have users refer to the ciphers talked about in the LUKS guide which is already web linked.